src/docs/patterns/tables.mdx

```diff
@@ -0,0 +1,469 @@
+
+<ImageGrid
+    images={[HomePagePreview, TableStackedAsList]}
+    captionsUnderImages={[
+        `On a crowded page, show a preview of the table content, with the option to view the full table in a dedicated page.`,
+        `Stack each property into a list item, with the ability to click into a list item to view the full item details.`,
+    ]}
+/>
+
+## Pagination
+
+If tables become necessary in your mobile application, pagination is especially helpful to reduce the load of both the server and the client-side render.
+
+<ImageGrid
+    images={[MobilePagination]}
+    captionsUnderImages={[
+        `Pagination is often used to reduce the page load. If not conflicting with other gestures, consider also adding left / right swipes to go to the previous / next page.`,
+    ]}
+    gridContainerProps={{ style: { justifyContent: 'flex-start' } }}
+/>
+
+> Be careful using pagination with multi-selection, as it is unclear whether the selection gets preserved when the page number changes.
+
+## Native Mobile Gestures
+
+In native mobile applications, you may also consider embedding action gestures common for touch screens, such as long-press and swipe.
+
+<MaterialDesignDescription title={'Gestures'} url={`https://material.io/design/interaction/gestures.html`} />
+
+However, these gestures should never be the only way to perform an action. For example, if your application is able to delete a table row, users may be able to either (1) left-swipe to bring up a contextual menu with a delete option, or (2) click into the target row, which opens a new page that contains a delete button.
+
+## Landscape Mode
+
+If a full table view is desired on mobile devices, consider offering a landscape view so that more columns would be visible at once.
+
+<ImageGrid images={[LandscapeTable]} caption={`Use the landscape mode for viewing the full table.`} regularWidth />
+
+However, most applications are meant to be used mainly on the portrait mode. In this case, you need to call out the option.
+
+<ImageGrid
+    images={[InviteToLandscapeX, InviteToLandscapeCountdown]}
+    caption={`For example, you can use a chip button to "invite" users to the landscape view.`}
+/>
+
+<Divider />
+
+# Design Specifications
+
+<ImageGrid
+    images={[SpecSize, SpecTypography, SpecPagination]}
+    captionsUnderImages={[`Dimensions`, `Colors and typography`, `Pagination`]}
+    regularWidth
+/>
+
+<MaterialDesignDescription
+    title={'Data Tables'}
+    description={`For additional instructions, refer to Material Design's design specs.`}
+    url={`https://material.io/components/data-tables#specs`}
+/>
+
+<Divider />
+
+# Developers
+
+Use the following components to implement this pattern:
+
+**Angular** and **Ionic**:
+
+-   Angular Material
+    -   [Table](https://material.angular.io/components/table/overview)
+
+**React**:
+
+-   Material UI
+    -   [Table](https://mui.com/components/tables/)
```

Comment:
Might also want to include a link to the XDataGrid: https://mui.com/components/data-grid/